modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -20,7 +20,9 @@
 import static java.util.concurrent.CompletableFuture.completedFuture;
 import static java.util.concurrent.CompletableFuture.failedFuture;
 import static org.apache.ignite.internal.util.ExceptionUtils.withCause;
+import static org.apache.ignite.lang.ErrorGroups.Common.UNKNOWN_ERR;
```

Comment:
why UNKNOWN_ERR is deprecated? im not sure that it is good pattern to use deprecated value, but if it is ok, the annotation should be removed